dashboard/config/scripts/levels/Pixelation - Lesson 14 - Fix bit offset v.2.level

```diff
@@ -0,0 +1,15 @@
+<Pixelation>
+  <config><![CDATA[{
+  "game_id": 44,
+  "created_at": "2015-08-25T16:30:00.000Z",
+  "level_num": "custom",
+  "user_id": 0,
+  "properties": {
+    "instructions": "Something happened to the bits in this image! Find error and fix it! Hint: a few extra bits were added. You should only need to remove some bits.",
```

Comment:
I realize you just copied this text and it's the same everywhere, but should it be "Find the error" not "Find error"?

(or maybe it's a meta joke? I did once implement a honeybadger-like thing with the word "error" misspelled everywhere)